crawl-ref/source/acquire.cc

```diff
@@ -549,7 +549,7 @@ static int _acquirement_missile_subtype(bool /*divine*/, int & /*quantity*/)
 
     switch (skill)
     {
-    case SK_SLINGS:    result = MI_SLING_BULLET; break;
+	case SK_SLINGS:    result = MI_STONE; break;
     case SK_BOWS:      result = MI_ARROW; break;
     case SK_CROSSBOWS: result = MI_BOLT; break;
```

Comment:
From a technical perspective, this all looks fine, with the exception of this tab; as with before, please change it to four spaces. I'd also advise you set your text editor/IDE/whatever you use for editing to indent with spaces instead of tabs.